### PR TITLE
Using attrib.exe to Manage States

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #130 ([QA] Control WinVFS with Attrib.exe)

Add a description how to use `attrib` to bulk manage file states when using VFS.

Backport to 4.0 and 4.1

### CODE REVIEW

**Comment by michaelstingl** [comment]
@TheOneRing @erikjv Could we get your review please?

File: modules/ROOT/pages/vfs.adoc [hunk 1]
```diff
@@ -142,3 +143,28 @@ image::vfs/vfs-free-up-space.png[Free Up Space, width=400,pdfwidth=70%]
 . When done, Explorer will show the file in _Placeholder_ state.
 +
 image::vfs/vfs-setup-successful-explorer.png[Setup Successful Explorer, width=600,pdfwidth=70%]
+
+## Using attrib.exe to Manage States
+
+For experienced users, xref:vfs-states[VFS states] can be managed also via the command line using `attrib`. This is particular useful if you want to change the state of a complete folder/subfolder. Open a `PowerShell` command line window like via the start menu and change to the directory of the ownCloud sync folder/subfolder that contains the data you want to bulk change.
```
Comment (EParzefall) [inline modules/ROOT/pages/vfs.adoc:149]:
```suggestion
Experienced users can also manage xref:vfs-states[VFS states] via the command line using `attrib`. This is particularly useful if you want to change the state of a complete folder/subfolder. Open a `PowerShell` command line window like via the start menu and change to the directory of the ownCloud sync folder/subfolder that contains the data you want to bulk change.
```

File: modules/ROOT/pages/vfs.adoc [hunk 2]
```diff
@@ -142,3 +143,28 @@ image::vfs/vfs-free-up-space.png[Free Up Space, width=400,pdfwidth=70%]
...
+
+Use the following command for help and to adapt switches according your needs.
```
Comment (EParzefall) [inline modules/ROOT/pages/vfs.adoc:151]:
```suggestion
Use the following command for help and to adapt switches according to your needs.
```

File: modules/ROOT/pages/vfs.adoc [hunk 3]
```diff
@@ -142,3 +143,28 @@ image::vfs/vfs-free-up-space.png[Free Up Space, width=400,pdfwidth=70%]
...
+----
+
+* Make a set of files or folders always available including subfolders:
++
+[source,powershell]
+----
+attrib -U +P .\* /D /S
+----
+
+* Make a set of files or folders online only including subfolders:
```
Comment (EParzefall) [inline modules/ROOT/pages/vfs.adoc:165]:
```suggestion
* Make a set of files or folders only online available, including subfolders:
```

**Review by EParzefall — APPROVED** [review]

**Comment by mmattel** [comment]
@TheOneRing @erikjv ping

**Review by TheOneRing — APPROVED** [review]
LGTM

**Pull request merged**